src/objective-c/GRPCClient/private/GRPCCore/GRPCHost.m

```diff
@@ -21,17 +21,16 @@
 #import <GRPCClient/GRPCCall+Cronet.h>
```

Comment:
This adds a cronet dependency - is that ok?

Reply:
This is ok. `GRPCCall+Cronet.h` is not dependency to Cronet. It's internal things related to Cronet.